tests/tests_lite/plugins/precision/test_native_amp_integration.py

```diff
@@ -67,6 +67,6 @@ def after_backward(self, model):
     ],
 )
 def test_native_mixed_precision(accelerator, precision, expected_dtype):
-    lite = NativeMixedPrecisionBoringLite(accelerator=accelerator, precision=16)
+    lite = NativeMixedPrecisionBoringLite(accelerator=accelerator, precision=precision)
```

Comment:
Including a small fix here. This was failing locally on the rtx card where bf16_cuda is available (not here in CI). And the test was obviously meant to take the parametrized argument as input.